cfgov/jinja2/v1/sublanding-page/index.html

```diff
@@ -53,6 +53,10 @@
             {{ render_block.render(block, loop.index) }}
         {% endif %}
     {%- endfor %}
+
+    {% if flag_enabled('EMAIL-POPUP', request) %}
```

Comment:
Typo: Should be `EMAIL_POPUP` (with an underscore, not a hyphen).

Reply:
fixed in https://github.com/cfpb/cfgov-refresh/pull/3484/commits/4aabd3c7c07f2bc6db6befbff83f37f1c6915eb3